colony/commands/sb.py

```diff
@@ -117,25 +120,61 @@ def do_end(self):
     def do_start(self):
         # get commands inputs
         blueprint_name = self.input_parser.sandbox_start.blueprint_name
+
         branch = self.input_parser.sandbox_start.branch
         commit = self.input_parser.sandbox_start.commit
-        name = self.input_parser.sandbox_start.sandbox_name
+        CommandInputValidator.validate_commit_and_branch_specified(branch, commit)
+
+        sandbox_name = self.input_parser.sandbox_start.sandbox_name
         timeout = self.input_parser.sandbox_start.timeout
+        wait = self.input_parser.sandbox_start.wait
         duration = self.input_parser.sandbox_start.duration
         inputs = self.input_parser.sandbox_start.inputs
         artifacts = self.input_parser.sandbox_start.artifacts
+        repo = get_and_check_folder_based_repo(blueprint_name)
+        self._update_missing_artifacts_and_inputs_with_default_values(artifacts, blueprint_name, inputs, repo)
+
+        with ContextBranch(repo, branch) as context_branch:
+            # TODO move error handling to exception catch (investigate best practices of error handling)
+            if not context_branch:
+                return self.error("Unable to start Sandbox")
```

Comment:
can we add here a more meaningful error message for the user?

Reply:
I think this is an old code that should have been removed as I don`t see a reason for context_branch to be None